eventhandler.go

```diff
@@ -166,7 +166,9 @@ func (eventHandler *EventHandler) handleNetworkEvent(event *Event) {
 	if !isPrivateIPAddress(event.IPAddress) &&
 		strings.Compare(event.IPAddress, "::1") != 0 &&
```

Comment:
This line `strings.Compare(event.IPAddress, "::1")` is redundant and can be removed